lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1559,17 +1559,26 @@ sub _failure_reason {
 
     my @failed_modules;
     my $modules = $self->modules;
+
     while (my $m = $modules->next) {
         if ($m->result eq FAILED || $m->result eq SOFTFAILED) {
-            push(@failed_modules, $m->name . ':' . $m->result);
+            # Look for serial failures which have bug reference
```

Comment:
That would not really be limited to "serial failures" so I guess you can get rid of this comment

Reply:
It would, because as of now `record_resultfile` with custom message is called only in case of serial failure.